README.md

```diff
@@ -6,23 +6,17 @@
 
 A simple JS library that detects mobile devices.
 
-## Why use isMobile?
+## Usage
 
-You probably shouldn't use this library unless you absolutely have to. In most cases, good [responsive design](https://en.wikipedia.org/wiki/Responsive_web_design) solves the problem of controlling how to
```

Comment:
It might sound silly, but I would actually prefer to continue convincing people to avoid user agent sniffing unless absolutely necessary. 😃

Reply:
Yeah good catch- deleting everything in the readme file was an accident. I just reverted and rebased with only updated files